### PR TITLE
Include friend search result #928

### PR DESCRIPTION
**Pull request by Bilb**
Include friend is search results from the message searchbox

Fixes https://github.com/loki-project/session-desktop/issues/928

### CODE REVIEW

File: ts/state/ducks/search.ts [hunk 1]
```diff
@@ -215,15 +215,6 @@ async function queryConversationsAndContacts(
     }
   }
 
-  // Inject synthetic Note to Self entry if query matches localized 'Note to Self'
```
Comment (neuroscr) [inline ts/state/ducks/search.ts]: why was this removed?

Comment (Bilb) [reply ts/state/ducks/search.ts]: because I thought that was not something we keep. I will correct it
